src/Common/src/Interop/Windows/kernel32/Interop.LoadLibraryEx.cs

```diff
@@ -4,14 +4,15 @@
 
 using System;
 using System.Runtime.InteropServices;
-
 using Microsoft.Win32.SafeHandles;
 
 internal partial class Interop
 {
     internal partial class Kernel32
     {
+    	public const int LOAD_LIBRARY_AS_DATAFILE = 0x00000002;
```

Comment:
Where is this being used? And it seems you have this field twice in your sources.

Reply:
its used in eventloginternal file